source/reference/minio-mc/mc-support-diagnostics.rst

```diff
@@ -0,0 +1,131 @@
+==========================
+``mc support diagnostics``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support diagnostics
+
+.. mc:: mc support diag
+
+Description
+-----------
+
+The :mc-cmd:`mc support diagnostics` command generates a health report for a MinIO deployment.
+For deployments registered with the MinIO subscription network, |subnet|, the command can automatically upload the health report for analysis.
```

Comment:
:thinking: so this renders as `the MinIO subscription network, MinIO SUBNET`

Which is a little heavy handed.

Easy way to fix is to add to `default-conf.py` under `rst_prolog` a new line

```
.. |subnet-short| replace:: `SUBNET <https://subnet.min.io?ref/docs>`__
```

Then sections like this become `For deployments registered with the MinIO Subscription Network (|subnet-short|)...`

Up to you

Reply:
💡 nice idea